packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -316,13 +322,20 @@ - (void)sendInitialized {
     if ([self duration] == 0) {
       return;
     }
-
+      
+      // Fetching the natural size and applying the preferredTransform
+      AVPlayerItemTrack *track = [self.player currentItem].tracks.firstObject;
+
+      CGSize naturalSize = track.assetTrack.naturalSize;
+      CGAffineTransform prefTrans = track.assetTrack.preferredTransform;
+      CGSize realSize = CGSizeApplyAffineTransform(naturalSize, prefTrans);
+      
     _isInitialized = true;
     _eventSink(@{
       @"event" : @"initialized",
       @"duration" : @([self duration]),
-      @"width" : @(width),
-      @"height" : @(height)
+      @"width" : @(fabs(realSize.width) ? : width),
```

Comment:
This change is also parts of your other PR.

Reply:
> This change is also parts of your other PR.

@saurus closed another.